loan.py

```
@@ -0,0 +1,26 @@
+    Args:
+    interest_rate: 0.10
+    amount_borrowed:10000
+    loan_term_years: 2
+
+    Returns:
+    Monthly payment amount
+    """
+    # Convert annual interest rate to monthly
+    monthly_rate = 1000
+    # Total number of monthly payments
+    number_of_payments = 24 
```

Ruff (0.12.2)

Local variable `number_of_payments` is assigned to but never used

Remove assignment to unused variable `number_of_payments`

(F841)